.github/workflows/build-test-package.yml

```diff
@@ -140,7 +140,6 @@ jobs:
         set(dashboard_cache "
         VTK_DIR:PATH=\${CTEST_DASHBOARD_ROOT}/VTK-build
```

Comment:
Once set during ITK build, this might not be needed here.

Reply:
Will do the changes. The build failed because it requires VTK::InteractionWidgets, VTK::InteractionStyle and VTK::RenderingOpenGL2. I am testing this now locally.